docs/README.md

```diff
@@ -107,7 +107,7 @@ environment variables as parameters:
 
 ```sh
 ( set -o pipefail
-curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/main/download.sh |
+curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/v0.3.1-beta.1/download.sh |
```

Comment:
It is not working. You can fix it, when we tag 0.4.0

Reply:
I am also OK to have this PR open and merge it just before 0.4.0 release.